content/docs/apps/govcloud.md

```diff
@@ -37,7 +37,7 @@ As part of [cloud.gov achieving FedRAMP authorization](https://18f.gsa.gov/2016/
 
 #### Who this affects
 
-Any systems running in the AWS East/West version of cloud.gov.
+Any systems running in the AWS East/West version of cloud.gov. To find all of the apps you have access to, visit https://dashboard.cloud.gov. All apps that aren't moved by [TODO] will be deleted, so make sure that you or someone on your team [coordinates]({{< relref "docs/help.md" >}}) with the cloud.gov team to ensure your migration is completed as smoothly as possible.
```

Comment:
Will this specify whether the app is in East/West rather than GovCloud? I don't notice it on first glance at the Dashboard if so, but I don't have much running there. I'd love to give people a specific thing to check about the apps that are listed if possible.